Mathlib/MeasureTheory/Measure/AddContent.lean

```diff
@@ -0,0 +1,184 @@
+/-
+Copyright (c) 2024 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne, Peter Pfaffelhuber
+-/
+import Mathlib.MeasureTheory.SetSemiring
+
+/-!
+# Additive Contents
+
+An additive content `m` on a set of sets `C` is a set function with value 0 at the empty set which
+is finitely additive on `C`. That means that for any finset `I` of pairwise disjoint sets in `C`
+such that `⋃₀ I ∈ C`, `m (⋃₀ I) = ∑ s ∈ I, m s`.
+
+Mathlib also has a definition of contents over compact sets: see `MeasureTheory.Content`.
+A `Content` is in particular an `AddContent` on the set of compact sets.
+
+TODO: refactor `Content` to use properties of `AddContent`.
```

Comment:
Is that really possible? compact sets don't form a semiring of sets, right?

Reply:
Right. Content is an AddContent, but if I only prove properties of AddContent on semi-rings of sets I can't use any of that to deduce properties of Content since compacts are not a semi-ring of sets. I'll just remove that todo.